ligo/README.md

```diff
@@ -1,5 +0,0 @@
-<!--
-SPDX-FileCopyrightText: 2020 tqtezos
-SPDX-License-Identifier: MIT
--->
-# stablecoin LIGO implementation
```

Comment:
I would still have README here btw :man_shrugging: 

At least a stub like this one for now.

Reply:
😲

Reply:
So what about restoring README?